config/routes.rb

```diff
@@ -1,8 +1,30 @@
+# Rails.application.routes.draw do
```

Comment:
You should delete this commented lines :wink:

Reply:
Done!